### PR TITLE
Core team notes 2023-12-08

### PR DESCRIPTION
**Pull request by tyler-potyondy**
### Pull Request Overview

Core team notes 12/8/23. 

### Testing Strategy

N/A

### TODO or Help Wanted

N/A

### Documentation Updated

- [ x ] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [ x ] Ran `make prepush`.

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2023-12-08.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2023-12-08.md:79]: Perhaps clean up the trailing blank lines in this file?

**Review by jrvanwhy — DISMISSED** [review]

File: doc/wg/core/notes/core-notes-2023-12-08.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- Alyssa: It makes sense to me that there would be a prebuilt copy optimized for size and a prebuilt copy optimized for space.
+- Alyssa: One other question, would inlining mirror require rebuilding?
+- Hudson: I do not think so. 
+- Alyssa: One other thought: unpacking large result objects often results in poor codegen even with inlining. Perhaps mirror inlining may help with this.
+- Hudson: I cannot remember if we turned this on in Tock.
+
+## TickV Discussion (https://github.com/tock/tock/discussions/3709)
+- Andrew: Just as a heads up, some of the agenda emails have been going to junk.
+- Andrew: I have been focusing on TickV and key value for my team.
+- Andrew: TickV spec states that a limitation is fragmentation. If something is written to a region and never erased, the flash will show as filled up even if they have been erased.
```
Comment (AndrewImwalle) [inline doc/wg/core/notes/core-notes-2023-12-08.md:29]: The wording of this is a little misleading. I was referring to the situation where something is written to a region in flash memory and is never removed. In such cases, even if the rest of the region is cleared, the flash memory might still indicate that the region is full, despite having only one valid entry. The current implementation of garbage collect is unable to solve this problem, which leads to the flash filling up while only having a few valid entries.

Comment (tyler-potyondy) [reply doc/wg/core/notes/core-notes-2023-12-08.md]: Went ahead and added this to the notes. Let me know if this is sufficient now and better captures your meaning.

Comment (AndrewImwalle) [reply doc/wg/core/notes/core-notes-2023-12-08.md]: That looks great! Thank you!

**Review by jrvanwhy — APPROVED** [review]

**Review by bradjc — APPROVED** [review]

**Pull request merged**